tests/test_core.py

```diff
@@ -1536,3 +1550,65 @@ def test_gpu_searchsorted():
 def test_client_to_func():
     with pytest.raises(NotImplementedError):
         core._client_to_func(core)
+
+
+def test_fix_isconstant_isfinite_conflicts():
+    T = np.full(12, 0.0, dtype=np.float64)
+    nan_indices = [1, 5, 9]
+    for idx in nan_indices:
+        T[idx] = np.nan
+
+    m = 3
+
+    n = len(T) - m + 1
+    T_subseq_isconstant = np.full(n, 1, dtype=bool)
+
+    ref = T_subseq_isconstant.copy()
+    ref[~core.rolling_isfinite(T, m)] = False
+
+    comp = core.fix_isconstant_isfinite_conflicts(T, m, T_subseq_isconstant)
+
+    npt.assert_almost_equal(ref, comp)
+
+
+def test_find_incompatible_args():
+    # case1: having exact required argument
+    def func_case1(x, y):
+        return
```

Comment:
This function and other functions below it are designed to test the functionality of `core._find_incompatible_args`. However, since we do not call these functions (right?), these functions are skipped according to the result shown in code coverage. Any suggestion @seanlaw ?

Reply:
@seanlaw 
FYI: To fix code coverage, I added `# pragma: no cover` here and for the next few functions.